### ISSUE TITLE
don't report unknown/any errors when casting

### ISSUE THREAD

**Issue by DetachHead** [DetachHead/basedpyright#84]
```py
from typing import cast

class Foo:
    value: asdfasdf = 1

cast(int, Foo().value) # error: unknown
```
related: #82
